packages/botonic-plugin-flow-builder/src/content-fields/flow-video.tsx

```diff
@@ -0,0 +1,22 @@
+import { Video } from '@botonic/react'
+import React from 'react'
+
+import { VideoNode } from '../flow-builder-models'
+import { ContentFieldsBase } from './content-fields-base'
+
+export class FlowVideo extends ContentFieldsBase {
+  public src = ''
+  public code = ''
+
+  static fromHubtypeCMS(component: VideoNode, locale: string): FlowVideo {
+    const newVideo = new FlowVideo(component.id)
+    newVideo.code = component.code
+    newVideo.src = this.getVideoByLocale(locale, component.content.video)
+    return newVideo
+  }
+
+  toBotonic(index: number): JSX.Element {
+    // @ts-ignore
+    return <Video key={index} src={this.src} />
```

Comment:
using the index as a key is not the best idea. It should be unique if it is a single string. Especially if it is a component rendered in a list with a map function. [keeping-list-items-in-order-with-key](https://es.react.dev/learn/rendering-lists#keeping-list-items-in-order-with-key)

Reply:
I agree @Iru89 , I'll change it in the next PR 👌